controller/handler_ctrl/inspect.go

```diff
@@ -42,10 +41,14 @@ func (h *inspectHandler) HandleReceive(request *channel2.Message, ch channel2.Ch
 
 	inspectRequest := &ctrl_pb.InspectRequest{}
 	if err := proto.Unmarshal(request.Body, inspectRequest); err != nil {
-		log.Errorf("unexpected error (%s)", err)
-		h.respondWithError(ch, request, fmt.Errorf("failed to decode request: (%v)", err).Error())
+		log.WithError(err).Error("failed to unmarshal inspect message")
```

Comment:
I see that handleInspect does reply but this if block simply returns and doesn't send a response.

So there is no one that was waiting for the response?